runtime/onert/core/include/ir/train/TrainableOperation.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_TRAIN_TRAINABLE_OPERATION_H__
+#define __ONERT_IR_TRAIN_TRAINABLE_OPERATION_H__
+
+#include "ITrainableOperation.h"
+
+namespace onert
+{
+namespace ir
+{
+namespace train
+{
+
+class TrainableOperation : public virtual ITrainableOperation
```

Comment:
I think it would be better to remove the keyword, `virtual` unless `ITrainableOperation` is going to be diamond inheritance.

Reply:
right, I see. The only common class in `ITrainableOperation` (as described in https://github.com/Samsung/ONE/blob/77d76ded8c10e3ef7187e0466a75a5ee541f1fc7/runtime/onert/core/include/ir/train/ITrainableOperation.h#L31-L33)